py/README.md

```diff
@@ -0,0 +1,42 @@
+# pydeck-earthengine-layers
+
+Pydeck wrapper for use with Google Earth Engine
+
+## Install
+
+```bash
+pip install pydeck-earthengine-layers
+```
+
+This also ensures [pydeck](https://pydeck.gl/) is installed. If you haven't
+previously enabled pydeck to be used with Jupyter, follow [its
+instructions](https://pydeck.gl/installation.html) to install.
+
+## Using
+
+For an example, see the
+[`examples/Introduction.ipynb`](examples/Introduction.ipynb) Jupyter Notebook.
```

Comment:
As noted in #26 , this link should be an absolute link to the raw file hosted on Github, because this README is also posted on the [PyPI project page](https://pypi.org/project/pydeck-earthengine-layers/), and the relative link doesn't work there.